lib/grpc_web/server/rack_app.rb

```diff
@@ -22,7 +22,18 @@
 # TODO: in-mem cache the result of lazy initialization
 class GRPCWeb::RackApp < ::Rack::Builder
   def handle(service_or_class, &lazy_init_block)
-    service_class = service_or_class.is_a?(Class) ? service_or_class : service_or_class.class
+    if service_or_class.is_a?(Class)
+      service_class = service_or_class
+    elsif service_or_class.is_a?(Module)
+      if lazy_init_block.nil?
+        raise(ArgumentError, "#{service_or_class} is an abstract interface. You must also provide an initializer block.") # rubocop:disable Layout/LineLength
+      end
+
+      service_class = service_or_class::Service
```

Comment:
I'm starting to wonder if we actually need to make this change on this gem, or if the feature we want to allow is already possible.
What if the caller were to use `service_or_class::Service` instead of `service_or_class` in the first place? So for example calling `handle` with `HelloService::Service`.
If a lazy init block isn't passed in, then it won't be caught in validation, but you'd get a `GRPC::Unimplemented` error when trying to call any methods on the service.